tests/output/test_io_mesh.py

```diff
@@ -31,9 +31,11 @@ def uniform_mesh(request):
                         extrusion_type="uniform", name=mesh_name)
 
 
-@pytest.fixture(params=["circlemanifold",
-                        "icosahedron",
-                        "cubedsphere"])
+@pytest.fixture(params=[
+    "circlemanifold",
+    pytest.param("icosahedron", marks=pytest.mark.broken),
```

Comment:
This wasn't broken before?

Reply:
Hmmm...

Reply:
This breakage seems to have magically gone away, maybe as a result of https://github.com/firedrakeproject/firedrake/pull/3730 ?

Reply:
That's my favourite kind of bug!